### ISSUE TITLE
the `elisp` folder is missing

### ISSUE THREAD

**Issue by xiuliren** [milkypostman/dotemacs#2]

**Comment by milkypostman** [comment]
yeah that's cause it's kinda custom thing I don't actually use anymore. It's just where I was putting pymacs at one point. Need to update the repo at some point I guess.
